.github/workflows/integration-tests.yml

```diff
@@ -44,6 +44,11 @@ on:
         description: 'Upper boundry for chain test'
         required: true
         default: '100000'
+      bob_min_port:
```

Comment:
I think there is no need to ask the user to set this parameter, since it does not affect the test in significant way